### PR TITLE
feat: Add Archway spec

### PR DESCRIPTION
**Pull request by MichaelLLC**
# Description

Closes: #XXXX

---

## Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] read the [contribution guide](https://github.com/lavanet/lava/blob/main/CONTRIBUTING.md)
* [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title, you can find examples of the prefixes below:
    
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] targeted the `main` branch
* [ ] provided a link to the relevant issue or specification
* [ ] reviewed "Files changed" and left comments if necessary
* [ ] included the necessary unit and integration tests
* [ ] updated the relevant documentation or specification, including comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] confirmed all CI checks have passed

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic, API design and naming, documentation is accurate, tests and test coverage

### CODE REVIEW

File: cookbook/specs/archway.json [hunk 1]
```diff
@@ -0,0 +1,1058 @@
...
+    "proposal": {
+        "title": "Add Specs: Archway",
+        "description": "Adding new specification support for relaying Archway data on Lava",
+        "specs": [
+            {
+                "index": "ARCHWAY",
+                "name": "archway mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDK"
```
Comment (Yaroms) [inline cookbook/specs/archway.json:11]: archway is running cosmos v0.50
needs to inherit accordingly

File: cookbook/specs/archway.json [hunk 2]
```diff
@@ -0,0 +1,1058 @@
...
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 1,
+                "average_block_time": 5500,
+                "allowed_block_lag_for_qos_sync": 2,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
```
Comment (Yaroms) [inline cookbook/specs/archway.json:22]: set the amount to 
15000000000ulava

File: cookbook/specs/archway.json [hunk 3]
```diff
@@ -0,0 +1,1058 @@
...
+                                    {
+                                        "expected_value": "archway-1"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
+                                    {
+                                        "latest_distance": 14400
```
Comment (Yaroms) [inline cookbook/specs/archway.json:341]: missing pruning verification for archive extension

File: cookbook/specs/archway.json [hunk 4]
```diff
@@ -0,0 +1,1058 @@
...
+                                    {
+                                        "expected_value": "archway-1"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
+                                    {
+                                        "latest_distance": 5760
```
Comment (Yaroms) [inline cookbook/specs/archway.json:904]: missing pruning verification for archive

File: cookbook/specs/archway.json [hunk 5]
```diff
@@ -0,0 +1,1058 @@
...
+                                    }
+                                ]
+                            }
+                        ],
+                        "extensions": [
+                            {
+                                "name": "archive",
+                                "cu_multiplier": 5,
+                                "rule": {
+                                    "block": 5680
```
Comment (Yaroms) [inline cookbook/specs/archway.json:914]: rule block is incorrect

File: cookbook/specs/archway.json [hunk 6]
```diff
@@ -0,0 +1,1058 @@
...
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 1,
+                "average_block_time": 5500,
+                "allowed_block_lag_for_qos_sync": 2,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
```
Comment (Yaroms) [inline cookbook/specs/archway.json:981]: set min stake to 15000000000ulava

File: cookbook/specs/archway.json [hunk 7]
```diff
@@ -0,0 +1,1058 @@
...
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+    },
+    "deposit": "10000000ulava"
```
Comment (Yaroms) [inline cookbook/specs/archway.json:1057]: deposit should be
1750000000ulava

File: cookbook/specs/archway.json (hunk 2 above)
Comment (MichaelLLC) [reply cookbook/specs/archway.json:22]: Fix

File: cookbook/specs/archway.json (hunk 7 above)
Comment (MichaelLLC) [reply cookbook/specs/archway.json:1057]: Fixed

File: cookbook/specs/archway.json (hunk 6 above)
Comment (MichaelLLC) [reply cookbook/specs/archway.json:981]: Fixed

File: cookbook/specs/archway.json (hunk 5 above)
Comment (MichaelLLC) [reply cookbook/specs/archway.json:922]: Please tell me where I can find the block number?

File: cookbook/specs/archway.json (hunk 3 above)
Comment (MichaelLLC) [reply cookbook/specs/archway.json:341]: How can I specify that the archive node must be started without pruning?
```
{
    "extension": "archive",
    "latest_distance": 6000000
}
```

File: cookbook/specs/archway.json (hunk 4 above)
Comment (MichaelLLC) [reply cookbook/specs/archway.json]: > pruning value is incorrect

Where can I get the recommended value?

**Pull request closed without merging**